route53_pca.tf

```diff
@@ -0,0 +1,32 @@
+# ------------------------------------------------------------------------------
+# Resource records that support the Continuous Phishing Campaign Assessment
+# (Con-PCA) application.
+# ------------------------------------------------------------------------------
+
+# ------------------------------------------------------------------------------
+# Production
+# ------------------------------------------------------------------------------
+
+resource "aws_route53_record" "pca_production_NS" {
+  provider = aws.route53resourcechange
+
+  name    = "con-pca.cool.${aws_route53_zone.cyber_dhs_gov.name}"
+  ttl     = 30
```

Comment:
This is a pretty low TTL for production.  Is there a particular reason why you want it this low?

Reply:
That's the value that @zenine07 gave me- perhaps he can let us know if there was a particular reason for that.

Reply:
There's not a particular reason it's that low, it was just the default value. Probably can increase it to 86400 and have a ttl of one day.

Reply:
I think that's a good idea for production

Reply:
Thanks - see https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/34/commits/f9f93f5de9e03713b1f5fe5ec6363de63f9c6b23